ext/standard/tests/streams/bug60106-002.phpt

```diff
@@ -0,0 +1,41 @@
+--TEST--
+Bug #60106 (stream_socket_server with abstract unix unix socket paths)
+--SKIPIF--
+<?php
+if( substr(PHP_OS, 0, 3) == "WIN" )
```

Comment:
It doesn't seem like either macOS nor FreeBSD support abstract socket names. So you should likely skip this test for everything but Linux.

Reply:
Yes, I forgot about that. I'm sad that OSX/FBSD don't support it though.